BridgeEmulator/default-config.json

```
@@ -583,6 +583,13 @@
         "mqttPort": 1883,
         "mqttServer": "mqtt",
         "mqttUser": ""
+    },
+    "homeassistant": {
+      "enabled": false,
+      "homeassistantIp": "127.0.0.1", 
```

```suggestion
      "homeAssistantIp": "127.0.0.1", 
```
Should probably be changed to capital A to be consistent with the other keys below (also change usages of this key in code)

---

Thanks - fixed.